Changes.md

```diff
@@ -38,6 +38,7 @@ _TBD_
 - [#3597](https://github.com/medic/medic-webapp/issues/3597): Update fonts to Noto.
 - [#3561](https://github.com/medic/medic-webapp/issues/3561): Display required fields when creating a "restricted to a place" user.
 - [#2522](https://github.com/medic/medic-webapp/issues/2522): Percentage target values are confusing.
+- [#2753](https://github.com/medic/medic-webapp/issues/2753): Update status time for messages and reports automatically.
```

Comment:
Is this correct?

Reply:
It won't make it in to 2.14.0 as we've already branched to `2.14.x`. We usually generate release notes just before release so you can just leave this out for now.